tests/positive/files/file.go

```diff
@@ -23,6 +23,7 @@ func init() {
 	register.Register(register.PositiveTest, CreateFileOnRoot())
 	register.Register(register.PositiveTest, UserGroupByID_2_0_0())
 	register.Register(register.PositiveTest, UserGroupByID_2_1_0())
+	register.Register(register.PositiveTest, AppendToAFile())
```

Comment:
Would be good to add tests covering appending to a file that doesn't exist & appending a file in a directory that doesn't exist.